pegasus/routes/v2_user_routes.rb

```diff
@@ -3,7 +3,11 @@
   dont_cache
   forbidden! unless dashboard_user
   content_type :json
-  JSON.pretty_generate(dashboard_user.slice_keys(:id, :name))
+  result = dashboard_user.slice_keys(:id, :name, :admin)
+  result[:owned_sections] = DASHBOARD_DB[:sections].
```

Comment:
Sadly, `dashboard_user` here is merely a `Hash`, the raw row pulled from the database, as pegasus does not have access to the `User` model in dashboard.  I could wrap getting owned sections into a utility somewhere though.